gemrb/plugins/GUIScript/PythonConversions.cpp

```diff
@@ -32,13 +32,13 @@ Color ColorFromPy(PyObject* obj) {
 		Color c;
 		// use _signed_ version of Clamp because PyInt_AsLong may return -1 on error and we would like this to = 0
```

Comment:
right, didn't even notice it, was just making it compile. It is true though, the function could return -1 on error and it's better to be safe. I'll force push an update.